content/iOS/Practices/iOS-practices.md

```diff
@@ -0,0 +1,118 @@
+# A few iOS practices that can go a long way when maintaining your projects.
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team who writes about the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
```

Comment:
🌫️ 🌫️ 🌫️

Reply:
Lol 
Wasn’t this the default header for articles ?

Reply:
I don't know haha, it's the first time I'm seeing it.

Reply:
me too xD